R/QFeatures-class.R

```diff
@@ -1175,12 +1336,62 @@ setMethod("updateObject", "QFeatures",
 ##' @export
 dropEmptyAssays <- function(object, dims = 1:2) {
     stopifnot(inherits(object, "QFeatures"))
-    if (!all(dims %in% 1:2))
+    if (!all(dims %in% 1:2)) {
         stop("Argument 'dims' must be in '1:2'.")
-    if (1 %in% dims)
+    }
+    if (1 %in% dims) {
         object <- object[, , nrows(object) > 0]
-    if (2 %in% dims)
+    }
+    if (2 %in% dims) {
         object <- object[, , ncols(object) > 0]
-    if (!length(object)) return(QFeatures())
+    }
+    if (!length(object)) {
+        return(QFeatures())
+    }
     object
 }
+
+
+##' @rdname QFeatures-class
+##' @param type `character(1)` that defines the type of the QFeatures.
+##'     The type can be either "bulk" or "scp" (default is "bulk"),
+##'      see [validQFeaturesType] for accepted types.
+##'
+##' @export
+##'
+setQFeaturesType <- function(object, type = "bulk") {
```

Comment:
May be not set a default `type` to force us to be explicit? What do you think @cvanderaa ?